### ISSUE TITLE
Feature Request: (Time,Position) Data for past wars.

### ISSUE THREAD

**Issue by TauZeph** [clapfoot/warapi#41]
Position data of  players for past wars, so we could recreate battles and flow of people on the maps. 

Imagine a data set like this:
For t= 0:30minutes:WarEndTime
Player#|Pos|Affiliation|

Player# would just be something to track each time a distinct individual joins. eg. At the beginning of the war, 140 people join a map, they get numbers 1 to 140. A player leaves and a new one joins, the new one is assigned 141. So on.

**Comment by QQ-q** [comment]
Before you read my comment, Hayden and I vividly recommend you to take this suggestion to the official discord server because the devs don't often check this thread. 

I'm not a developer but being someone who uses this API and has seen the changes and talked with peeps who have used the foxhole api for a long time, such as Hayden. I honestly doubt this will ever happen for 3 main reasons:

**1.** All of the features of the API are based on one principle: anyone, in either warden or colonial faction, can access this information, no matter their location. These includes pretty much every single icon **BUT** FOBs and pretty much any structures that doesn't need a CV to be built. If this hasn't been cleared enough, let me give you an example:
When you open your map in the home region, you can click in any map and know which towns have been taken and which not. Everyone has access to this information, so we can considered this to be public.  (_This is why one of the endpoints is dynamic/public_) However you do not have access to faction locked intel, such as friendly position or FOBs location since only 1 faction has access to this kind of information. Thus, not making it public.

**2.**  A few months we used to have access to each map population but the devs removed it so adding this feature will contradict the devs' decision. 

**3.** Your idea, although really cool, it has some critical problems. If you can trace each individual position, you can easily know which town is the least defended which allows for easier pushes that normally shouldn't happen. Also, in one word, **spies**. 

Just because of these doesn't mean it will never happen, all you need to do is refine your idea and make sure it fits with the overall purpose of this API. Feel free to contact me through discord!!

**Comment by TauZeph** [comment]
@QQ-q 

Thank you for your input. I reiterate that this is for past wars - Ie. wars that have already been completed.  This means that all points (1,2,3) in your reply are not applicable.

**Comment by QQ-q** [comment]
Yeah, my bad then. I didn't read your suggestion properly and for that I apologize.
